cmd/internal/cli/actions.go

```diff
@@ -154,7 +154,16 @@ func handleShub(u string) (string, error) {
 	imageName := uri.GetName(u)
 	imagePath := cache.ShubImage("hash", imageName)
 
-	libexec.PullShubImage(imagePath, u, true, noHTTPS)
+    exists, err := cache.ShubImageExists("hash", imageName)
```

Comment:
Hey @marcelm , the code looks good to me. Can you make sure you run `go fmt ./...` with the `go1.11` version of `go` installed and then push that as a new commit? Your code looks to be incorrectly formatted, and `go` is very particular about formatting. Cheers, looking forward to reviewing once that's fixed :)

Reply:
Hi @bauerm97,

  Thanks!  I pushed a new version of this fix with the correct formatting.  

-Mike